### DISCUSSION TITLE
`dvc metrics show` provides an empty output

### DISCUSSION THREAD

**Discussion by aidneML** [treeverse/dvc#7868] [Help]
I have setup a pipeline which completes successfully after running `dvc repro`.
I have several `.json` files which I've confirmed to be filled with metric values, specified under the `metrics:` section in `dvc.yml`.

However, the output of `dvc metrics show` is empty.
Running `dvc metrics show --verbose` results in the following:
```
$ dvc metrics show --verbose
2022-06-09 20:06:27,376 DEBUG: Analytics is enabled.
2022-06-09 20:06:27,990 DEBUG: Trying to spawn '['daemon', '-q', 'analytics', 'C:\\Users\\...\\AppData\\Local\\Temp\\tmpxi13lof4']'
2022-06-09 20:06:28,009 DEBUG: Spawned '['daemon', '-q', 'analytics', 'C:\\Users\\...\\AppData\\Local\\Temp\\tmpxi13lof4']'
$
```

It seems that a daemon is spawned but returns immediately without providing any meaningful response.
What could be the problem with my setup?

- Python version : 3.9.6
- OS: Windows 10

Thanks!

**Comment by daavoo** [comment]
Hi @aidneML ! Could you share the output of `dvc doctor` and `dvc metrics show -vv` (double `v` to enable TRACE log).

In addition, if you run the command passing the path to one of those files `dvc metrics show {your_json_file}` , does it show anything?

**Reply by aidneML** [reply]
Hi @daavoo! 

Thank you for the quick reply!

I'll start with the most relevant output first (probably):

```
$ dvc metrics show evaluation_train.json
WARNING: 'evaluation_train.json' was not found in current workspace.
```

Humm, it seems I don't have the file in my workspace, even though it is tracked by git?
I'm probably missing something obvious here :D

Output of `dvc doctor`:
```
$ dvc doctor
DVC version: 2.11.0 (pip)
---------------------------------
Platform: Python 3.9.6 on Windows-10-10.0.19042-SP0
Supports:
        webhdfs (fsspec = 2022.5.0),
        http (aiohttp = 3.8.1, aiohttp-retry = 2.4.6),
        https (aiohttp = 3.8.1, aiohttp-retry = 2.4.6)
Cache types: hardlink
Cache directory: NTFS on C:\
Caches: local
Remotes: azure
Workspace directory: NTFS on C:\
Repo: dvc, git
```

**Reply by aidneML** [reply]
Update:

If I try to explicitly add the `.json` file to my DVC workspace, I get the following:

```
$ dvc add evaluation_train.json
ERROR: output 'evaluation_train.json' is already specified in stage: 'evaluate_train'.
```

**Reply by dberenbaum** [reply]
@aidneML Sorry for the late reply. Could you try `dvc metrics show -vv evaluation_train.json` and post the output?
